.travis.yml

```diff
@@ -88,7 +88,6 @@ jobs:
 cache:
   directories:
     - node_modules
-    - build-system/tasks/e2e/node_modules
```

Comment:
I think this requires the additional step of deleting the existing Travis cache.

Reply:
It should be ok, Travis detects if there are files that need to be deleted.
https://travis-ci.org/ampproject/amphtml/jobs/550474957#L2019